src/test/java/com/moodybluez/enterprise/EnterpriseApplicationTests.java

```diff
@@ -76,27 +77,27 @@ private void whenEntryIsCompleted() {
 
 		DayOfWeek dayOfWeek = DayOfWeek.from(localDate);
 
-		Entry entry = new Entry();
 		entry.setMoodID(moodID);
-		entry.setReasonDesc(reasonForMood);
+		entry.setDescription(reasonForMood);
 		entry.setDate(new Date());
-		entry.date.setDate(entryDate);
-		entry.date.setDayOfWeekID(dayOfWeek.getValue());
-		entry.date.setDayOfWeekDesc(dayOfWeek.name());
-		entry.date.setDayOfWeekID(dayOfWeek.getValue());
+		entry.setEntryID(1);
+		entry.setWeekDayID(dayOfWeek.getValue());
+		entry.getDate().setDate(entryDate);
+		entry.getDate().setWeekDayID(dayOfWeek.getValue());
+		entry.getDate().setWeekDay(dayOfWeek.name());
 
-		entryService.saveEntry(entry);
+		entryDAO.saveEntry(entry);
 	}
 
 	private void thenReturnMoodEntry() throws Exception {
-		Map<String, Entry> moodEntries = entryService.fetchAll();
+		Map<Integer, Entry> moodEntries = entryDAO.fetchAll();
 		boolean moodEntryPresent = false;
 		for (Map.Entry mapElement : moodEntries.entrySet()) {
-			String date = (String) mapElement.getKey();
+			int entryID = (int) mapElement.getKey();
 			Entry entry = (Entry) mapElement.getValue();
 
-			if (entry.getMoodID() == 3 && entry.getReasonDesc() == "I laid in bed all day."
-					&& date == "2/22/2021") {
+			if (entry.getMoodID() == 3 && entry.getDescription().equals("I laid in bed all day.")
```

Comment:
It is recommended to use .equals method for string comparison rather than == which is mostly used for other var types.
Citation - "**== checks if both objects point to the same memory location whereas .equals() evaluates to the comparison of values in the objects**' - [GeeksForGeeks](https://www.geeksforgeeks.org/difference-equals-method-java/)